### ISSUE TITLE
Update color of Login banner with SUL-oriented color

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/universalviewer#31]
(Broken out from #24)

Change the background color of the information banner to something in the Stanford color palette (and maybe some with less contrast, so the lazy, unauthenticated user can use the viewer without closing the banner and not be too distracted by it).

Cardinal red was discussed and looks very Stanfordy, though it was brought up that it might not have enough contrast with the adjacent brown top bar.

**Comment by edsilv** [comment]
You should just need to add a `@brand-warning` variable with the desired colour to your theme: https://github.com/sul-dlss/uv-en-GB-theme/blob/master/variables.less

**Comment by mejackreed** [comment]
## UV Default
<img width="469" alt="screen shot 2017-08-01 at 9 19 58 am" src="https://user-images.githubusercontent.com/1656824/28835560-9b123654-769a-11e7-8040-ab177d354553.png">

## EAAB00 "Sun"
<img width="472" alt="screen shot 2017-08-01 at 9 23 10 am" src="https://user-images.githubusercontent.com/1656824/28835689-159288b6-769b-11e7-8ddf-8df2650f07e4.png">

## EFC040 Derived tint from Sun
<img width="462" alt="screen shot 2017-08-01 at 9 56 38 am" src="https://user-images.githubusercontent.com/1656824/28836981-066d1626-76a0-11e7-88f2-0db649fc187f.png">

## 007c92 "Lagunita"
<img width="469" alt="screen shot 2017-08-01 at 10 00 02 am" src="https://user-images.githubusercontent.com/1656824/28837052-3d622edc-76a0-11e7-9262-ff801fc1243f.png">

## 8d3c1e "Redwood"
<img width="466" alt="screen shot 2017-08-01 at 10 00 40 am" src="https://user-images.githubusercontent.com/1656824/28837070-4e0465de-76a0-11e7-96ad-da6bc58dd534.png">

## 175e54 "Palo Alto"
<img width="466" alt="screen shot 2017-08-01 at 10 01 12 am" src="https://user-images.githubusercontent.com/1656824/28837091-6186397a-76a0-11e7-8151-791038b807d4.png">

**Comment by ggeisler** [comment]
I had made a minor argument for a color that jumps out less than the original default color, just to make it easier for users who might ignore the banner to continue using the viewer. From that point of view, Redwood works in that the contrast to the background is less sharp than the other options.

However, I don't feel too strongly about this. If we went with any of the last three I think we might need to also change the close "X" and the button to white. Are those colors also easily overridable? 

If not, I like the Sun and the variant of that; they wouldn't require any changes to the button and close icon. It's just a bit harder to focus on the viewer when the stronger color banner is visible.

Maybe @jvine has a firmer opinion to help us make a decision.

**Comment by mejackreed** [comment]
@ggeisler yes we can easily change the X and other styling/button colors

**Comment by jvine** [comment]
Looking at it in the context of a black and white photo is maybe misleading, too - any colour we choose is going to interfere with some images. (Speaking of, are we sticking with the dark brown frame? I've really come to hate it.) That said:

- we have typically used Cardinal for things relating to authentication
- I've been thinking of the non-logged-in state as a preview that doesn't have much utility in itself, so a really obvious "this isn't everything; you need to login" banner seems appropriate - but maybe that's not realistic?
- I find Redwood depressing
- I like Sun and Lagunita

How's that for firm?

**Comment by ggeisler** [comment]
@jvine I thought you didn't like Cardinal in this content? If that is not an issue, that makes the most sense to me. Otherwise I'd vote for Sun or the variant of it.

I had created another ticket to deal with the text on the banner (related to your second bullet) but it was closed: https://github.com/sul-dlss/universalviewer/issues/32. I think it was closed due to a resolution of how the string is configurable, but my intent was to have a separate ticket to deal with the wording we want to present on the banner. So I guess we could create a new ticket for that.

**Comment by mejackreed** [comment]
Reopened #32 @ggeisler .. I feel like if we want to make those modifications, feel free to make a suggestion there. This will be done in stacks.

**Comment by mejackreed** [comment]
Thanks for the feedback!

So here I have "Dark Red" implemented. Imagine the close "X" is white

<img width="633" alt="screen shot 2017-08-03 at 1 59 14 pm" src="https://user-images.githubusercontent.com/1656824/28943536-055343f4-7854-11e7-9f1b-853b38a83490.png">

And here "Cardinal Red"

<img width="574" alt="screen shot 2017-08-03 at 2 00 20 pm" src="https://user-images.githubusercontent.com/1656824/28943565-21163eb6-7854-11e7-94b9-799833ed5690.png">

Thoughts on the final direction we should move?

**Comment by ggeisler** [comment]
If I was picking I'd go with the dark red, but it's pretty much a toss-up to my eye.

**Comment by jvine** [comment]
Agree - dark red, but it's pretty close.

**Comment by mejackreed** [comment]
👍

**Comment by mejackreed** [comment]
Now waiting on https://github.com/UniversalViewer/universalviewer/pull/490 to see if that gets merged

**Comment by mejackreed** [comment]
Resolved in https://github.com/sul-dlss/uv-en-GB-theme/commit/4a62503d86686cea789e649f4aed0841c7fb83a1
